synchronization/src/androidTestCyface/java/de/cyface/synchronization/SyncAdapterTest.java

```diff
@@ -64,7 +80,8 @@
     private Context context;
     private ContentResolver contentResolver;
     private Account account;
-    private SyncAdapter oocut;
+    private SyncAdapter objectUnderTest;
+    private AccountManager accountManager;
 
     @Before
     public void setUp() {
```

Comment:
Nicht wundern, ich setze die Version in den Test auf der 2. Stelle hoch wenn die Tests vorher flaky waren weil die Accounts nicht in allen Fällen (z.B. wenn man einen Test stoppt, teils sogar wenn einer failed) entfernt wurden.

Das ist für mich keine kleine Änderung.